### DISCUSSION TITLE
Questions about printing

### DISCUSSION THREAD

**Discussion by Stong1234** [Sakati84/3DPrintColorizer#100] [Q&A]
Amazed by the idea and got my first successful print. However, I hav 2 questions would like to seek help.

1. I am using ender 3v2. Seems I can only set Z speed max at 10.  How can I set it higher ?

2. The sharpie get dried out after 4-5 hours.  Any good suggestion to keep it from drying?

thanks a lot for all your help
Happy color printing!

**Comment by csanii123** [comment]
1 - I think you need to use a custom firmware to increase the Z speed max above 10.
2 - Not sure what can be done here other than to optimize the speeds and make sure you selected interlace colors that way it only color every other layer - You may also want to look at CHEP's fast printing profiles. I haven't tried for coloring prints but for regular prints it significantly speeds up printing. The other thing to consider is if you have 4 or 5 colors but some colors are not used till much later in printing then leave caps on and then when it gets close to using those pens remove the caps.
